.prettierrc

```diff
@@ -0,0 +1,10 @@
+{
+  "arrowParens": "always",
+  "bracketSpacing": true,
+  "endOfLine": "auto",
+  "printWidth": 120,
```

Comment:
Just out of curiosity, the line width is 120 characters, but I can see very long lines in your code, are they matching this rule?

Reply:
Sorry to ask that, but could point out an example? To be honest, the cases that I noticed that this rule was not being applied were related to comments, but it seemed to be working fine with code in general. 😕

Reply:
I'm reviewing on GitHub and they really mess the tab character giving the wrong impression. But there are some examples with trailing comments and strings that are overflowing the 120 character limit.

Reply:
That's true... 😕 I noticed that the rules that I set on `prettier` were not working with comments, and I saw that there is an [issue on prettier's repo about this](https://github.com/prettier/prettier/issues/265), but I confess that I ended up putting it aside... 😅 

Anyways, that is indeed something that annoys me a little bit as well, and I could absolutely try to fix it if you want me to 😄